### ISSUE TITLE
[GTK] Draw pen does not affect the `text` color

### ISSUE THREAD

**Issue by hiiamboris** [red/red#4901]
**Describe the bug**
```
r1: rtd-layout ["ABC"]
r1/color: blue        ;) doesn't affect anything
r2: rtd-layout ["ABC"]
r2/font: make font! [color: blue]
view [base cyan draw compose/deep [pen magenta fill-pen yellow text 10x10 "ABC" text 10x30 (r1) text 10x50 (r2)]]
```
Result:

|Windows|GTK|
|-|-|
|![](https://i.gyazo.com/f662efbaae88b487cae7f1be9ac5957a.png) | ![](https://i.gyazo.com/34725f34a2d7acbae1012037ceb49a4d.png) |

So:
1. Windows ignores font color, uses `pen` only
2. GTK ignores pen, uses `font/color` only, and also draws a white background for no reason

**Expected behavior**

GTK works as Windows backend

**Platform version**
```
Red 0.6.4 for Windows built 12-May-2021/6:12:03+03:00  commit #cebd5b9
Red 0.6.4 for Linux built 11-May-2021/6:20:25+03:00  commit #81d0416
```

**Comment by qtxie** [comment]
> Windows ignores font color, uses pen only

I think it should use the font color as it's more specific.

**Comment by hiiamboris** [comment]
That IMO would be bad (as bad as current rich-text design) because to change the color I would have to change the font object (which is shared for obvious reasons), plus mind all the on-change triggers it may incur.

**Comment by qtxie** [comment]
> because to change the color I would have to change the font object

No, you don't have to. It just takes the priority when you set it.

**Comment by hiiamboris** [comment]
Should rich-text's face/color set the background too then, when not `none`?

**Comment by qtxie** [comment]
If face/size is set, yes.

**Comment by hiiamboris** [comment]
If not set, then for the single line.
